entwatch/ze_solgryn_v1_3.cfg

```diff
@@ -0,0 +1,98 @@
+"entities"
+{
+	"0"
+	{
+		"name"              "Ammo"
+		"shortname"         "Ammo"
+		"color"             "{lightblue}"
+		"buttonclass"       "func_button"
+		"filtername"        "ammouser"
+		"hasfiltername"     "true"
+		"blockpickup"       "false"
+		"allowtransfer"     "true"
+		"forcedrop"         "true"
+		"chat"              "true"
+		"hud"               "true"
+		"hammerid"          "4250026"
+		"mode"              "2"
+		"maxuses"           "0"
+		"cooldown"          "60"
+		"maxamount"         "1"
+	}
+	"1"
+	{
+		"name"              "Rocket Launcher"
+		"shortname"         "Rocket"
+		"color"             "{orange}"
+		"buttonclass"       "func_button"
+		"filtername"        "rocketuser"
+		"hasfiltername"     "true"
+		"blockpickup"       "false"
+		"allowtransfer"     "true"
+		"forcedrop"         "true"
+		"chat"              "true"
+		"hud"               "true"
+		"hammerid"          "4148012"
+		"mode"              "2"
+		"maxuses"           "0"
+		"cooldown"          "5"
+		"maxamount"         "1"
+	}
+	"2"
+	{
+		"name"              "Suge Knight"
```

Comment:
In case someone is asking why it's named like that:
https://www.youtube.com/watch?v=Q3keX35f66Y